Firmware/temp_model.h

```diff
@@ -59,6 +59,7 @@ struct model_data
 };
 
 static bool enabled;          // model check enabled
+static bool valid = false;    // model is valid
```

Comment:
Why is this needed? temp_model::calibrated() should do the same and avoid having to set/reset another state variable?